parm/config/gefs/config.fcst

```diff
@@ -107,19 +107,10 @@ if (( gwd_opt == 2 )); then
     export launch_level
 fi
 
-# Sponge layer settings
```

Comment:
Probably want to leave this comment, no?

Reply:
@aerorahul you are right.  I will add this line back